src/main/java/com/google/devtools/build/lib/standalone/StandaloneModule.java

```diff
@@ -51,13 +54,20 @@ public void executorInit(CommandEnvironment env, BuildRequest request, ExecutorB
             env.getBlazeWorkspace().getBinTools(),
             testTmpRoot);
 
+    CoreOptions coreOptions = env.getOptions().getOptions(CoreOptions.class);
+    boolean runfilesEnabled = BuildConfiguration.runfilesEnabled(coreOptions);
+    // True if all runfile trees will be created when building binaries/test. If so, we won't need
+    // the RunfilesTreeUpdater.
+    boolean createRunfilesAtBuild = BuildConfiguration.shouldCreateRunfilesTree(coreOptions);
```

Comment:
This will not work if someone changes the value of this flag in a configuration transition. Sure, it would be kind of weird to do so, but one can do that from Starlark (same in `WorkerModule`). You'll need to find a way to convey that bit from the configured target itself.

Reply:
Interesting point. Would you consider this a blocker for this change or can this be done as an immediate follow up?

Reply:
Blocker. Let's not submit known-incorrect code.